tests/test_main.py

```diff
@@ -29,10 +29,10 @@ def read_latest(live, metric_name):
 def test_logging_no_step(tmp_dir):
     dvclive = Live("logs")
 
-    dvclive.log_metric("m1", 1)
+    dvclive.log_metric("m1", 1, plot=False)
     dvclive.make_summary()
 
-    assert not (tmp_dir / "logs" / "m1.tsv").is_file()
+    assert not (tmp_dir / "logs" / "plots" / "metrics" / "m1.tsv").is_file()
```

Comment:
why changing the path here?

Reply:
The test was outdated and shouldn't have been passing. `tmp_dir / "logs" / "m1.tsv"` is never going to be generated in the current dvclive version. So I adjusted the path to make it fail properly and then added the flag to make it pass again.